packages/bull-vault/src/BullStrategy.sol

```diff
@@ -111,7 +111,22 @@ contract BullStrategy is ERC20, LeverageBull, UniBull {
         IERC20(usdc).transfer(msg.sender, usdcBorrowed);
     }
 
-    function withdraw() external {}
+    function withdraw(uint256 _crabAmount) external {
+        uint256 share = _crabAmount.wdiv(totalSupply());
```

Comment:
Is this always true? `share = crabAmount/totalCrabSupply` and `share = bullAmount/totalBullSupply`, but I don't think `share = crabAmount/totalBullSupply`.

Reply:
Ah this is wrong var name, it should be `bullAmount`